### ISSUE TITLE
Study Guide - Remove implicit prerequisites

### ISSUE THREAD

**Issue by stephanschielke** [Retronator/Pixel-Art-Academy#95]
Just a low prio suggestion:

To me, it doesn't make sense to have requirements for a new study if another requirement already contains it.
Example:
![image](https://user-images.githubusercontent.com/401815/110218293-d18db100-7eb0-11eb-94f0-bf951bf965e9.png)

"Detailed Outlines" has the requirements:
* 2D Shapes
* Base Shading Sketch

However "Base Shading Sketch" already has "2D Shapes" as a requirement.

Now you end up with a net of crossing lines between studies.

I think it makes sense to remove all implicit dependencies that were already met by another study requirement.

**Comment by MatejJan** [comment]
Yeah, I agree on this, it's there to allow further alternative activities where you'd get base shading sketch separately from the 2D shapes. 

Everything now also has a bit of hidden complexity since some tasks inside an activity have their own requirements. So for example, the first task of Detailed outlines requires only '2D shapes', it's only the last one that also needs 'base shading sketch'. So you could complete the first few tasks as soon as you know 2D shapes. This isn't made clear however (unless you hover over the tasks), as the prerequisites just appear all grouped down below. 

It's not ideal, no, far from it. Some of this wiring mess will be solved once there actually is tracking of which interests you practiced. Then prerequisites under the task graph won't even show up (since you already have that interest acquired). But once you're planning for the first time, this will still be a problem. 

It's a perhaps over-engineered system where each task can have its own reqs. I am a fan of simplicity however so cutting down might happen if it turns out so much detail is not needed. For now I wanted to rather have too much info (model exactly what I think you need for each task). I think at this point I'd first wait to see how things behave once interest tracking is in and do any such decisions later. But at the same time I'm very open to discussing this further to get a wider range of opinions/ideas.
